clustered/ehcache-clustered/build.gradle

```diff
@@ -180,6 +184,31 @@ tasks.register('distDir', Sync) {
   with(distributions.main.contents)
 }
 
+var copyServer = tasks.register("copyServer") {
```

Comment:
This can be an explicit `Sync` task rather than a task that calls `project.sync(...)`.

Reply:
converted